pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -74,6 +75,13 @@ stdenv.mkDerivation rec {
     (replaceVars ./souploader.diff {
       nixLibSoup3Path = "${lib.getLib libsoup_3}/lib";
     })
+
+    (fetchpatch {
+      name = "musl.patch";
+      url = "https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/dd1fc2b7931f5789815e17dda2ef7c31b9fba563.patch";
+      stripLen = 2;
```

Comment:
Surprised this is enough without `extraPrefix = "";` to add `a/` and `b/` prefixes.

Reply:
It's because it leaves the "gst-plugins-good/" prefix, which functions the same as a/ and b/ would.